### ISSUE TITLE
Component not loading in Chrome, DOMContentLoaded not called.

### ISSUE THREAD

**Issue by atiro** [shakacode/react_on_rails#1330]
Have read through previous issues on the issue with defer which I think might be part of this, but haven't found quite the same situation. We have a component that is being registered fine and can be seen in debugging in both Chrome & Firefox, but it's not appearing in the page in Chrome. Tracking it down a little the problem is with (using 11.1.5, without Turbolinks):

```
  window.setTimeout(function () {
    if (!turbolinksInstalled() || !turbolinksSupported()) {
      if (document.readyState === 'complete') {
        debugTurbolinks('NOT USING TURBOLINKS: DOM is already loaded, calling re
actOnRailsPageLoaded');
        reactOnRailsPageLoaded();
      } else {
        document.addEventListener('DOMContentLoaded', function () {
          debugTurbolinks('NOT USING TURBOLINKS: DOMContentLoaded event, calling
 reactOnRailsPageLoaded');
          reactOnRailsPageLoaded();
        });
      }
    }
  });
```
in clientStartup.js. In Firefox the readyState is complete so reactOnRailsPageLoaded() is called, in Chrome the readyState is "interactive", so the DOMContentLoaded listener is added. But it doesn't seem to be called, so the component never appears.

Should reactOnRailPageLoaded be called straight away in both cases ? I'm not sure why Chrome doesn't call DOMContentLoaded after (presumably) readyState changes from interactive to complete, there seems to be a lot of contradictory views on what should be happening. 

Reading through the code for v12 this seems to have been changed a lot so it would probably fix things, but unfortunately we can't move to that at the moment.

**Comment by justin808** [comment]
Hi @atiro,  I haven't used turbolinks in a long time.

See https://github.com/shakacode/react_on_rails/commit/7b301f9e87b4f27b9197207f8e210c38095c551b#diff-dff9a544208442c275bbe37f37438219L268

I think I just fixed that.

Are you sure you can't upgrade?

If you're really stuck, I could backport this tiny change to 11.x (what would X be?)

**Comment by atiro** [comment]
Cheers @justin808 

Sadly the ruby version for 12 isn't something we can jump to at the moment. 

It would be great if you could release that for 11. A point release seems OK ?

**Comment by justin808** [comment]
I'm fixing here: https://github.com/shakacode/react_on_rails/pull/1331

**Comment by justin808** [comment]
@atiro Please try out https://rubygems.org/gems/react_on_rails/versions/11.3.1.beta.0, https://www.npmjs.com/package/react-on-rails/v/11.3.1-beta.0

**Comment by justin808** [comment]
I just released 11.3.1 for this issue.

**Comment by atiro** [comment]
Perfect, all good for me @justin808 

Many thanks for the quick response & the project
